lbrynet/core/HashBlob.py

```diff
@@ -78,14 +83,20 @@ class HashBlob(object):
     """A chunk of data available on the network which is specified by a hashsum"""
 
     def __init__(self, blob_hash, upload_allowed, length=None):
+        assert is_valid_blobhash(blob_hash)
         self.blob_hash = blob_hash
         self.length = length
         self.writers = {}  # {Peer: writer, finished_deferred}
         self.finished_deferred = None
-        self.verified = False
+        self._verified = False
         self.upload_allowed = upload_allowed
         self.readers = 0
 
+    @property
+    def verified(self):
+        # protect verified from being modified by other classes
+        return self._verified
```

Comment:
is this `@property` pattern the recommended thing to do in python? my Magic SenseTM is tingling